spec/spec_helper.rb

```diff
@@ -0,0 +1,111 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
+# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
+# The generated `.rspec` file contains `--require spec_helper` which will cause
+# this file to always be loaded, without a need to explicitly require it in any
+# files.
+#
+# Given that it is always loaded, you are encouraged to keep this file as
+# light-weight as possible. Requiring heavyweight dependencies from this file
+# will add to the boot time of your test suite on EVERY test run, even for an
+# individual file that may not need all of that loaded. Instead, consider making
+# a separate helper file that requires the additional dependencies and performs
+# the additional setup, and require it from the spec files that actually need
+# it.
+#
+# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+require 'byebug'
+require 'securerandom.rb'
+require 'faker'
+require File.dirname(__FILE__) + '/../lib/transferwise_client.rb'
+
+TransferwiseClient.configure do |config|
+  config.profile_id = '3431'
+  config.auth_key = '4a569d2b-264c-46ef-8280-7d8e96ac5b10'
```

Comment:
whoops, you accidentally added this key information into source control. 
Seeing that this is a public repo, these are now compromised.
We should get new keys from transferwise.

Reply:
No, thats ok. Its key for sandbox endpoint with my own account. I wanted to write some integration tests. But i think i will not write any integration tests and remove